realm/lib/realm-chat.hoon

```diff
@@ -661,6 +653,8 @@
           ['data' (mtd data.notif)]
           ['include_player_ids' a+(turn player-ids |=([id=@t] s+id))]
           ['headings' (contents title.notif)]
+          ['ios_badgeType' s+'SetTo']
+          ['ios_badgeCount' (numb unread-count)]
```

Comment:
I woulda left everything alone and then just added these two lines as your only change, but make this one be:
```hoon
['ios_badgeCount' (numb unread-count.mtd)]
```
and then yeah also remove line 683 like you did below.

Reply:
Alright! But `unread-count` doesn't seem accesible here? Your suggestion gives me:

```hoon
/lib/realm-chat/hoon::[665 35].[665 51]>
-find.unread-count.mtd
```

Reply:
['ios_badgeCount' (numb unread-count.data.notif)]